packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/PlanUpgradeDetails.tsx

```diff
@@ -28,12 +28,20 @@ export const PlanUpgradeDetails = ({
   invoicePreview: FirstInvoicePreview;
   className?: string;
 }) => {
-  const { config } = useContext(AppContext);
-  const { amount, currency, interval, interval_count } = selectedPlan;
+  const { navigatorLanguages, config } = useContext(AppContext);
+  const { product_name, amount, currency, interval, interval_count } =
+    selectedPlan;
+  const formattedInterval = formatPlanInterval(interval, interval_count);
```

Comment:
At the moment `formatPlanInterval` returns different values depending on 1 and non-1 `interval_count`. However the l10n string only supports values for the case where `interval_count` is 1. This could lead to a mismatch between the l10n string and the fallback text.

Reply:
Good catch! Fixed.